dvc/command/status.py

```diff
@@ -68,6 +69,12 @@ def run(self):
                 self._show(st, indent)
             elif not self.repo.stages:
                 logger.info(self.EMPTY_PROJECT_MSG)
+            elif self.args.cloud or self.args.remote:
+                logger.info(
+                    self.IN_SYNC_MSG.format(
+                        remote=self.repo.config["core"].get("remote")
```

Comment:
Notice that we have `--remote` CLI flag too, and config['core']['remote'] is the default one that might not be set  :slightly_smiling_face:

Reply:
Oh, good point, updated.